deps/v8/include/v8-platform.h

```diff
@@ -285,14 +288,23 @@ class Platform {
    */
   virtual bool OnCriticalMemoryPressure(size_t length) { return false; }
 
+  virtual int NumberOfWorkerThreads() {
+    return static_cast<int>(NumberOfAvailableBackgroundThreads());
+  }
+
   /**
    * Gets the number of threads that are used to execute background tasks. Is
    * used to estimate the number of tasks a work package should be split into.
    * A return value of 0 means that there are no background threads available.
    * Note that a value of 0 won't prohibit V8 from posting tasks using
    * |CallOnBackgroundThread|.
    */
-  virtual size_t NumberOfAvailableBackgroundThreads() { return 0; }
+  V8_DEPRECATE_SOON(
+      "NumberOfAvailableBackgroundThreads() is deprecated, use "
+      "NumberOfAvailableBackgroundThreads() instead.",
```

Comment:
This is supposed to say `NumberOfWorkerThreads()`, right? Not that it matters much, but the message might be … confusing? :smile:

Reply:
This typo is still present in V8 master, so should be fixed there first.